bitnami/spark/Chart.yaml

```
@@ -21,4 +21,4 @@ name: spark
 sources:
   - https://github.com/bitnami/bitnami-docker-spark
   - https://spark.apache.org/
-version: 4.0.0
+version: 4.0.1
```

As this is introducing new parameters, can you increase the minor version instead? (second number).

```suggestion
version: 4.1.0
```

---

Sure. I just oriented on the pod annotations, there it was a patch, as well.